servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/HttpOffloadingTest.java

```diff
@@ -232,50 +262,63 @@ void clientSettingsStreamIsOffloaded() throws Exception {
         assertNoAsyncErrors(errors);
     }
 
-    @Test
-    void clientCloseAsyncIsOffloaded() throws Exception {
-        subscribeTo(errors, httpConnection.closeAsync());
+    @ParameterizedTest(name = "offloadClose={0}")
+    @ValueSource(booleans = {false, true})
+    void clientCloseAsyncIsOffloaded(boolean offloadClose) throws Exception {
+        setup(offloadClose);
+        subscribeTo(httpConnection.closeAsync());
         terminated.await();
         assertNoAsyncErrors(errors);
     }
 
-    @Test
-    void clientCloseAsyncGracefullyIsOffloaded() throws Exception {
-        subscribeTo(errors, httpConnection.closeAsyncGracefully());
+    @ParameterizedTest(name = "offloadClose={0}")
+    @ValueSource(booleans = {false, true})
+    void clientCloseAsyncGracefullyIsOffloaded(boolean offloadClose) throws Exception {
+        setup(offloadClose);
+        subscribeTo(httpConnection.closeAsyncGracefully());
         terminated.await();
         assertNoAsyncErrors(errors);
     }
 
-    @Test
-    void clientOnCloseIsOffloaded() throws Exception {
+    @ParameterizedTest(name = "offloadClose={0}")
+    @ValueSource(booleans = {false, true})
+    void clientOnCloseIsOffloaded(boolean noOffloads) throws Exception {
+        setup(noOffloads);
```

Comment:
Would be nice to add a test that verifies adding a filter or connection factory that influences the close adds offloading. Could be done in a follow-up, not necessary to block this PR